app.js

```diff
@@ -13,6 +14,16 @@ app.post('/', (req, res) => {
   res.send("Hello! - Post request received. JS code running.");
 });
 
+app.get('/questions', (req, res) => {
+  // Access QS params via: req.query
+  // Mock question JSON accessed via variable: sourceQuestions
+  res.send({
+    message: "Hello! - GET request received. JS code running."
+  });
+});
+
 app.listen(port, () => {
   return console.log(`server is listening on http://localhost:${port}`);
-});
\ No newline at end of file
+});
+
+// RESTART APP ON CODE CHANGES AFTER SAVING FILE(s) !!
```

Comment:
Not sure how much help you want to give people using this, but you could just use wire up [Nodemon](https://nodemon.io/) and forego the shouting comment. 😉

Reply:
Nodemon would be my preference as well.

Reply:
can one of you PR it in? I have other story work I need to get back to.

Reply:
For now - I am limiting the number of deps required to as little as possible. Agree, Nodemon is superior to the comment, but not required.